packages/blocks/src/render-blocks.js

```diff
@@ -62,14 +62,34 @@ const parseNodes = ( nodes ) =>
 			name: toLower( node.tagName ),
 			children: parseNodes( node.childNodes ),
 			props: {
-				...attributes,
+				...parseTagProps( toLower( node.tagName ), attributes ),
 				className,
 				style: parseStyles( style ),
 				key: uniqueId( KEY_PREFIX ),
 			},
 		};
 	} );
 
+/**
+ * Parses DOM nodes into an array of objects to be used to create React elements.
+ *
+ * @param  {string} tagName
+ * @param {Object}  props object.
```

Comment:
Nit: can we align these? I actually thought our linter should be picky about that 🤔